composer.json

```diff
@@ -28,8 +28,8 @@
         "behat/mink-extension": "*",
         "behat/mink-goutte-driver": "*",
         "behat/mink-selenium2-driver": "*",
-        "dealerdirect/phpcodesniffer-composer-installer": "*",
         "drupal/coder": "8.3.1",
+        "dealerdirect/phpcodesniffer-composer-installer": "*",
```

Comment:
@navneet0693 This change doesn't actually do anything. Dependencies are generally listed in alphabetical order and composer will use the declared dependencies of packages to determine the download order.

Reply:
@Kingdutch yeah you are right, I tried changing order because of https://www.drupal.org/project/coder/issues/3106119#comment-13424507